backend/pick-git/src/main/java/com/woowacourse/pickgit/post/domain/comment/CommentContent.java

```diff
@@ -1,14 +1,36 @@
 package com.woowacourse.pickgit.post.domain.comment;
 
+import java.util.Objects;
+import javax.persistence.Column;
 import javax.persistence.Embeddable;
 import javax.persistence.Lob;
 
 @Embeddable
 public class CommentContent {
 
+    private static final int MAX_COMMENT_CONTENT_LENGTH = 100;
+
+    @Column(nullable = false, length = 100)
     @Lob
     private String content;
```

Comment:
```suggestion
    @Lob
    private String content;
```

`@Column`이랑 `@Lob` 동시에 적용 안 되는 걸로 알고 있어요.
nullable, length 체크는 따로 유효성 검증 로직 추가해서 해야 할 거 같습니다.
이에 따른 도메인 단위 테스트도 작성해주세요 !

Reply:
오오 맞네요 !! 다니 굳굳